web/src/api.ts

```diff
@@ -139,9 +140,9 @@ export const addListing = async (
  * Fetches all Listings.
  * @param queryParams Query parameters used to filter listings to retrieve.
```

Comment:
Update the comment when API signature is changed.

Reply:
@xlai20 It's the same though :o I only made changed the type to be more specific rather than accepting any filter string fields.